news/2020-10-22-beatmapping-world-championship-2020-concludes.md

```diff
@@ -0,0 +1,67 @@
+---
+layout: post
+title: "osu! Beatmapping World Championship 2020 Concludes!"
+date: 2020-10-22 12:00:00 +0000
+---
+
+After 5 months of intense competition among a total of 32 teams enlisted in this contest, the results are finally out! 
+
+![](/wiki/shared/news/banners/bwc2020.jpg)
+
+We're extremely happy to announce the conclusion of the 2020 edition of the **osu! Beatmapping World Championship! (o!bwc)**. This edition was full of surprises and turnarounds that no one in the entire galaxy would have ever expected! 
+
+From the elimination of the well-known Team France and Team China by the underdog Team Argentina, to the very close battle between Team Russia and Team South Korea, we can affirm that the show has never stopped! 
+
+If you wish to get a full rundown of the whole tournament brackets or take a look at the entries, all of those are available on [**our website**](https://obwc.net/) 
+
+Take a look at the VOD of the finals right below!
+
+<iframe width="100%" height="315" src="https://www.youtube.com/embed/VxRbdb9EUkY" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+
+![o!BWC 2020 Podium](/wiki/shared/news/2020-10-22-beatmapping-world-championship-concludes/podium.png)
```

Comment:
if possible, it'd be great to change `Alphabet` -> `Aistre` because of namechance a while back

Reply:
Waiting on kaetwo to fix it